src/components/AppRouting.js

```diff
@@ -7,7 +7,7 @@ import { useOrganizationState } from '@/providers/OrganizationProvider'
 export const AppRouting = ({ appName, defaultPath, appRoutes, children }) => {
   const { apps } = useOrganizationState()
 
-  const app = apps.filter(app => app.name === appName)
+  const [app] = apps.filter(app => app.name === appName)
```

Comment:
why was this change?

Reply:
This component redirects from `/<app>` to `/<app>/<first-app-address-found`, but it wasn't getting that app address correctly as `filter()` returns an array. On second thoughts, using ' find()'  instead looks cleaner. I'll change this